src/components/Tooltip/TooltipBox.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import ListPropType from '../../prop-types/list';
+import './TooltipBox.scss';
+
+const propTypes = {
+    children: PropTypes.oneOfType([
+        PropTypes.element,
+        PropTypes.string,
+    ]).isRequired,
+    status: ListPropType(['default', 'success', 'error']),
+    className: PropTypes.string,
+    style: PropTypes.objectOf(PropTypes.oneOfType([
+        PropTypes.string,
+        PropTypes.number,
+    ])),
```

Comment:
For consistency, could you please order props alphabetically here as well?

Reply:
Yup! 👍